plugin/health/README.md

```diff
@@ -21,6 +21,17 @@ a 503. *health* periodically (1s) polls plugin that exports health information.
 plugin signals that it is unhealthy, the server will go unhealthy too. Each plugin that
 supports health checks has a section "Health" in their README.
 
+More options can be set with this extended syntax:
+
+~~~
+health [ADDRESS] {
+    lameduck DURATION
+}
+~~~
+
+* Where `lameduck` will make the process unhealthy then *wait* for **DURATION** before the process
+  shuts down. If you want to explicitaly disable use `lameduck 0`.
```

Comment:
why would you need to explicitly disable? is it enabled by default?

Reply:
No. I'll delete this; obviously confusing.